include/psa/crypto_extra.h

```diff
@@ -33,13 +33,39 @@ extern "C" {
 #endif
 
 /* If the size of static key slots is not explicitly defined by the user, then
- * set it to the maximum between PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE and
- * PSA_CIPHER_MAX_KEY_LENGTH.
+ * set it to the maximum between PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE,
+ * PSA_CIPHER_MAX_KEY_LENGTH and PSA_MAC_MAX_SIZE.
```

Comment:
Please update this comment, which still mentions `PSA_MAC_MAX_SIZE`. It doesn't need to get into the details of which macros are relevant, it can just say something about checking the most common cases for various categories of keys.